### PR TITLE
added git to skills selection

### PR DESCRIPTION
**Pull request by limormek**

### CODE REVIEW

**Review by shem8 — APPROVED** [review]

File: steps/setup_docker_skills_selection.yml [hunk 1]
```diff
@@ -27,6 +27,8 @@ startFlow:
                     label: 'Communication'
                   - value: 'Documentation'
                     label: 'Documentation'
+                  - value: 'git'
```
Comment (esapir) [inline steps/setup_docker_skills_selection.yml:30]: upper case? All the other options are upper cased

Comment (limormek) [reply steps/setup_docker_skills_selection.yml:30]: I was actually wondering about it, since in the database the value is lowercase

Comment (esapir) [reply steps/setup_docker_skills_selection.yml:30]: @NivRichter ?

Comment (limormek) [reply steps/setup_docker_skills_selection.yml]: @esapir both work, I changed it 👍

**Pull request merged**